lgc/patch/PatchResourceCollect.cpp

```diff
@@ -1850,8 +1850,22 @@ void PatchResourceCollect::mapBuiltInToGenericInOut() {
       } else
         builtInUsage.vs.cullDistance = 0;
 
-      builtInUsage.vs.layer = false;
-      builtInUsage.vs.viewportIndex = false;
+      if (nextBuiltInUsage.layerIn) {
+        assert(nextInOutUsage.builtInInputLocMap.find(BuiltInLayer) != nextInOutUsage.builtInInputLocMap.end());
+        const unsigned mapLoc = nextInOutUsage.builtInInputLocMap[BuiltInLayer];
+        inOutUsage.builtInOutputLocMap[BuiltInLayer] = mapLoc;
+        availOutMapLoc = std::max(availOutMapLoc, mapLoc + 1);
+      } else
+        builtInUsage.vs.layer = false;
```

Comment:
We try to follow the rule that if one side of an if/else uses braces, then so should the other. So please add braces here.